www/js/metrics-factory.js

```diff
@@ -130,12 +154,6 @@ angular.module('emission.main.metrics.factory', ['emission.plugin.kvstore'])
         mets: 9.8
       }
     },
-    "IN_VEHICLE": {
```

Comment:
I would retain `IN_VEHICLE` as a backup in case the mode inference step has been excluded from the pipeline or it is not running properly. There are multiple layers of backup/robustness in the e-mission code (e.g. the display in the list/detail views continues to support the `speedometer` mode for `IN_VEHICLE` trips, so would be good to be consistent.

Reply:
Previously you had `IN_VEHICLE` as an average of "train" and "car". Would it make sense now to  average `TRAIN`, `CAR` and `BUS`, or would you have a different suggestion? 
e.g.
```
else if (mode == "IN_VEHICLE") {
    result += ((footprint[CAR] + footprint[BUS] + footprint[TRAIN]) / 3) * mtokm(userMetrics[i].values);
}
```